roles/ansible/tower/config-ansible-tower-ocp/tasks/setup_pvc.yml

```diff
@@ -0,0 +1,52 @@
+---
+- block:
+   - name: Ensure user is authenticated with OCP
+     shell: |
+       oc login {{ openshift_host }} \
+          --token {{ openshift_token }} \
+          --insecure-skip-tls-verify={{ openshift_skip_tls_verify | default(false) | bool }}
+     no_log: true
+
+   - name: Checking if target project exists on OCP..
+     command: oc get project {{ openshift_project }}
+     register: getProject
+     failed_when: false
+
+   - name: Creating target project..
+     shell: oc new-project {{ openshift_project }}
+     register: getProject
+     failed_when: false
+
+   - name: Creating target project..
+     shell: oc new-project {{ openshift_project }}
+     when: getProject is search("not found")
+
+   - name: Creating Temp Dir ..
+     file:
+      state: directory
+      path: "./temp"
+
+   - name: Source the PVC template ...
+     template:
+      src: pvc.j2
+      dest: "./temp/pvc.yml"
+
+   - name: Apply the PVC manifest..
+     shell: oc apply -f ./temp/pvc.yml
```

Comment:
Instead of a using a temp dir (which in this case isn't really a temp dir), I'd suggest just using the `lookup` module, e.g.:
```
echo -n '{{ lookup("template", "./src.j2") }}' | oc apply -n {{ openshift_project }} -f -
```

Reply:
I've changed the role according to suggestion